modules/nodes-cluster-enabling-features-install.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * nodes/cluster/nodes-cluster-enabling-features.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-cluster-enabling-features-install_{context}"]
+= Enabling feature sets at installation
+
+You can enable feature sets for all nodes in the cluster by editing the `install-config.yaml` file before you deploy the cluster.
+
+.Prerequisites
+
+* You have an `install-config.yaml` file.
+
+.Procedure
+
+. Use the `featureSet` parameter to specify the the name of the feature you want to enable. For example, `TechPreviewNoUpgrade`.
++
+.Sample `install-config.yaml` file with an enabled feature set
+
+[source,yaml]
+----
+compute:
+- hyperthreading: Enabled
+  name: worker
+  platform:
+    aws:
+      rootVolume:
+        iops: 2000
+        size: 500
+        type: io1
+      metadataService:
+        authentication: Optional
+      type: c5.4xlarge
+      zones:
+      - us-west-2c
+  replicas: 3
+featureSet: TechPreviewNoUpgrade
+----
+
+. Save the file and reference it when using the installation program to deploy the cluster.
+
+[NOTE]
+====
+Enabling the `TechPreviewNoUpgrade` feature set cannot be undone and prevents minor version updates. These feature sets are not recommended on production clusters.
+====
+
+.Verification
+
+You can verify that the feature gates are enabled by looking at the `kubelet.conf` file on a node after the cluster is installed.
```

Comment:
Interesting. I did not know that. You can also use `oc describe featuregate cluster`